website/src/config.js

```diff
@@ -0,0 +1,27 @@
+export const PROJECT_TYPE = 'github'; // 'github' || 'phab'
+
+export const PROJECT_NAME = 'viewport-mercator-project';
+export const PROJECT_ORG = 'uber-common';
+export const PROJECT_URL = `https://github.com/${PROJECT_ORG}/${PROJECT_NAME}`;
+export const PROJECT_DESC = 'Utilities for Web Mercator Projections';
+
+export const PROJECTS = {
```

Comment:
now I'm thinking we could even make a package with these projects or event a gist file that would be refreshed and updated automatically!

Reply:
Yep have had the same idea. There is also talk (especially with @jckr) about giving our repository suite an identity, e.g. its own repo with its own ocular docs describing all the component frameworks. 

It would be a great place to host such common files.

One reason this is a good idea is that we can't put all the unpromoted repos in the drop-down link, but it would be nice to have a place that mentions them all.